packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -70,6 +70,8 @@ export class MirrorNodeClient {
         DESC: 'desc'
     };
 
+    private static badGatewayHttpStatusCode = 502;
```

Comment:
Does it have to be one that is assigned?  I could see someone accessing mirrornode through a gateway and legitametly getting this.  Maybe 567 as it is obviously not a real server error?

Reply:
Doesn't have to be, just need to be consistent for metrics.
I'm updated